src/views/DrawerNavigatorItems.tsx

```diff
@@ -54,13 +67,24 @@ const DrawerNavigatorItems = ({
       const scene = { route, index, focused, tintColor: color };
       const icon = renderIcon(scene);
       const label = getLabel(scene);
-      const accessibilityLabel = typeof label === 'string' ? label : undefined;
+      const accessibleLabel =
+        typeof accessibilityLabel === 'string'
+          ? accessibilityLabel
+          : typeof label === 'string'
+          ? label
+          : undefined;
+      const accessibleHint =
```

Comment:
These check is not needed. Directly passing `accessibilityHint` will do the same thing